i18n/en.json

```diff
@@ -1992,6 +1992,7 @@
   "general_tab.codeTitle": "Invite Code",
   "general_tab.emptyDescription": "Click 'Edit' to add a team description.",
   "general_tab.getTeamInviteLink": "Get Team Invite Link",
+  "general_tab.membersManagedByGroups": "Members of this team are added and removed by linked groups. [Learn More]()",
```

Comment:
@wiersgallak Do we know the link URL for this yet?

Reply:
@wiersgallak As discussed I have removed the 'learn more' hyperlink until the docs are ready.